app/controllers/admin/rents_controller.rb

```diff
@@ -0,0 +1,83 @@
+class Admin::RentsController < ApplicationController
+  before_action :login_required
+  before_action :authenticate
+  before_action :set_rent, only: [:show, :update, :edit]
+  before_action :set_rents, only: [:edit, :new]
+  before_action :set_councils, only: [:new, :edit]
+
+
+  def main
+    @rents = Rent.ascending.from_date(Time.zone.now.beginning_of_day)
+    @rent_grid = initialize_grid(@rents)
+    @faqs = Faq.where(answer: '').where(category: 'Bil')
+  end
+
+  def show
+
+  end
+
+  def edit
+
+  end
+
+  def create
+    @rent = Rent.new_with_status(rent_params, nil)
+    respond_to do |format|
+      if @rent.save(validate: false)
+        format.html { redirect_to admin_rent(@rent),
+                                  notice: 'Bokningen skapades, mejl har skickats till angiven epostadress!' }
```

Comment:
Line is too long. [109/80]<br>Expression at 28, 109 should be on its own line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.